push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -61,7 +61,7 @@ public void sendMessagesToPushNetwork(@Observes @Dequeue MessageHolderWithTokens
         final UnifiedPushMessage unifiedPushMessage = msg.getUnifiedPushMessage();
         final Collection<String> deviceTokens = msg.getDeviceTokens();
 
-        logger.info(String.format("Received UnifiedPushMessage from JMS queue, will now trigger the Push Notification delivery for the %s variant (%s)", variant.getType().getTypeName(), variant.getVariantID()));
+        logger.info(String.format("UnifiedPushMessage was successfully received. Push Notification delivery for the {} variant ({}) will now be triggered.", variant.getType().getTypeName(), variant.getVariantID()));
```

Comment:
when at it, please remove the `String.format()` and just got w/ the vanilla `{}` syntax from slf4j. thx